introduction.md

```diff
@@ -2,4 +2,7 @@ I'm papayaman
 -The papayaman isn;t ripe yet
 -Once the papaya is good to go .. it'll be sweet and delicious
 
-Javascript
\ No newline at end of file
+*Javascript
+*HTML
+*PHP
+*ActionScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?